src/data/meetings.js

```diff
@@ -98,6 +104,7 @@ export default {
     showRoster: false,
     showSettings: false,
     cameraID: null,
+    videoPermission: 'UNKNOWN',
```

Comment:
use `null` instead of `'UNKNOWN'`, we removed that value from the interface https://github.com/webex/component-adapter-interfaces/blob/master/src/MeetingsAdapter.js#L95